From be9170dfa3bf54a425ff6d822e13fd74cbe75fec Mon Sep 17 00:00:00 2001
Message-ID: <be9170dfa3bf54a425ff6d822e13fd74cbe75fec.1713654571.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1713654571.git.sharlatanus@gmail.com>
References: <cover.1713654571.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 2 Apr 2024 15:31:21 +0100
Subject: [PATCH 16/47] gnu: python-jwst: Update to 1.14.0.

* gnu/packages/astronomy.scm (python-jwst): Update to 1.14.0.
[propagated-inputs]: Remove python-asdf-transform-schemas; add
python-importlib-metadata, python-packaging, python-pysiaf, and
python-synphot.

Change-Id: I0310e0680457a7851d07fa14406edf079000fcd5
---
 gnu/packages/astronomy.scm | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 6908be0694..374229bcf4 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3969,13 +3969,13 @@ (define-public python-jplephem
 (define-public python-jwst
   (package
     (name "python-jwst")
-    (version "1.13.4")
+    (version "1.14.0")
     (source (origin
               (method url-fetch)
               (uri (pypi-uri "jwst" version))
               (sha256
                (base32
-                "0n5cfjayg15kf5p6pns7k2zfcldqz1464ndbldgzanc33fp9wnfz"))))
+                "0knl2x0z3djsln8mi3a4bkxjz49iwmnr1v6wwsk13zvpkfq21vv4"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -3992,19 +3992,21 @@ (define-public python-jwst
     (propagated-inputs (list opencv
                              python-asdf
                              python-asdf-astropy
-                             python-asdf-transform-schemas
                              python-astropy
                              python-bayesicfitting
                              python-crds
                              python-drizzle
                              python-gwcs
+                             python-importlib-metadata
                              python-jplephem
                              python-jsonschema
                              python-numpy
+                             python-packaging
                              python-photutils
                              python-poppy
                              python-psutil
                              python-pyparsing
+                             python-pysiaf
                              python-requests
                              python-scikit-image
                              python-scipy
@@ -4014,6 +4016,7 @@ (define-public python-jwst
                              python-stpipe
                              python-stsci-image
                              python-stsci-imagestats
+                             python-synphot
                              python-tweakwcs
                              python-wiimatch))
     (native-inputs (list python-colorama
@@ -4022,6 +4025,7 @@ (define-public python-jwst
                          python-pytest-doctestplus
                          python-pytest-openfiles
                          python-requests-mock
+                         ;; python-ruff ; not packed yet in Guix
                          python-setuptools-scm))
     (home-page "https://jwst-pipeline.readthedocs.io/en/latest/")
     (synopsis
-- 
2.41.0

